template/amp/dfp-creative.html

```diff
@@ -1,17 +1,24 @@
-<script src = "https://cdn.jsdelivr.net/npm/prebid-universal-creative@0.4.0/dist/creative.js"></script>
+<script src = "https://acdn.adnxs.com/prebid/universal-creative/creative.js"></script>
 <script>
-var adId = "%%PATTERN:hb_adid%%";
-var cacheHost = "%%PATTERN:hb_cache_host%%";
-var cachePath = "%%PATTERN:hb_cache_path%%";
-var uuid = "%%PATTERN:hb_cache_id%%";
-var mediaType = "%%PATTERN:hb_format%%";
-var pubUrl = "%%PATTERN:url%%";
-var env = "%%PATTERN:hb_env%%";
-var size = "%%PATTERN:hb_size%%";
+  var ucTagData = {};
+  ucTagData.adServerDomain = "";
+  ucTagData.pubUrl = "%%PATTERN:url%%";
 
-try {    
-    pbjs.renderAd(document, adId, {cacheHost: cacheHost, cachePath: cachePath, uuid: uuid, mediaType: mediaType, pubUrl: pubUrl, env: env, size: size});
-} catch(e) {
+  // if not DFP, use these params
+  ucTagData.adId = "%%PATTERN:hb_adid%%";
+  ucTagData.cacheHost = "%%PATTERN:hb_cache_host%%";
+  ucTagData.cachePath = "%%PATTERN:hb_cache_path%%";
+  ucTagData.uuid = "%%PATTERN:hb_cache_id%%";
+  ucTagData.mediaType = "%%PATTERN:hb_format%%";
+  ucTagData.env = "%%PATTERN:hb_env%%";
+  ucTagData.size = "%%PATTERN:hb_size%%";
+
+  // if DFP, use this param
```

Comment:
Should we move this up (after line 5) ? Because most of the time adserver will be DFP.